test/core/formatError.test.js

```diff
@@ -60,4 +61,26 @@ suite("formatError", function() {
     var answer = Parsimmon.formatError(input, parser.parse(input));
     assert.deepEqual(answer, expectation);
   });
+
+  test("multi-line line-number padding", function() {
+    var parser = Parsimmon.seq(
+      Parsimmon.string("\n")
+        .many()
+        .then(Parsimmon.string("a"))
+    );
+    var expectation =
+      "\n" +
+      "-- PARSING FAILED --------------------------------------------------\n\n" +
+      "   8 | \n" +
+      "   9 | \n" +
+      "> 10 | b\n" +
+      "     | ^\n\n" +
+      "Expected one of the following: \n\n" +
+      "'\n', 'a'\n";
+
+    var input = new Array(9).join("\n") + "\nb";
+    var answer = Parsimmon.formatError(input, parser.parse(input));
+
+    assert.deepEqual(answer, expectation);
+  });
```

Comment:
Would you mind adding test cases for 3 and 4 digit line numbers? This is looking good but it would be nice to have.

Reply:
Thanks for the feedback, I'll include that kind of test asap.